### DISCUSSION TITLE
Desde que comecei a tentar rodar o projeto pelo Visual Studio comecei a ter problemas

### DISCUSSION THREAD

**Discussion by oimarcella** [balta-io/forum#599] [Tira Dúvidas] [answered]
Boa noite, estou no curso de Fundamentos de ASP.NET 6  e começando a criar o projeto Blog e usar o Visual Studio.

Tive problemas para rodar, pois eu abri a solução no Visual Studio, e ai surgiram outros arquivos com nome Blog.csproj, alguns de backup, e ficava dando erro para eu instalar ```Microsoft.EntityFrameworkCore.Design```. Acabei deletando os arquivos que tinham um formato diferente do que eu estava vendo nas aulas até então, e ai consegui instalar o pacote.

Agora fiz um clean, e em seguida um build e não está compilando, e retorna esse erro:
![image](https://github.com/balta-io/forum/assets/43733159/6a2090d5-b6e7-4620-966a-39d971649489)

**Comment by oimarcella** [comment, marked as answer]
Eu fiz a famosa "solução da net", fechar e abrir novamente, aí aproveitando também fiz mais um ```dotnet clean``` e  ```dotnet build``` e funcionou!

**Reply by andrebaltieri** [reply]
hahahahah, está no caminho certo 🤣
